src/Sarif.Sarifer/AnalyzeProjectCommand.cs

```diff
@@ -80,52 +80,14 @@ private void MenuCommandCallback(object caller, EventArgs args)
                     // Disable the menu click when we are analysing.
                     SariferPackageCommand.DisableAnalyzeCommands(this.menuCommandService);
 
-                    List<string> targetFiles = GetFiles(project);
+                    List<string> targetFiles = SariferPackageCommand.GetFiles(project);
 
                     this.backgroundAnalysisService.AnalyzeAsync(project.FullName, targetFiles, this.cancellationTokenSource.Token)
                         .FileAndForget(FileAndForgetEventName.BackgroundAnalysisFailure);
                 }
             }
         }
 
-        private static List<string> GetFiles(Project project)
```

Comment:
merged this in the SariferPAckageCommand that we are already sharing between the two commands